plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableQuery.php

```diff
@@ -353,7 +353,7 @@ private function date_to_date_query_arg( $date ): array {
 		} elseif ( ! is_a( $date, 'WC_DateTime' ) ) {
 			// For backwards compat (see https://github.com/woocommerce/woocommerce/wiki/wc_get_orders-and-WC_Order_Query#date)
 			// only YYYY-MM-DD is considered for date values. Timestamps do support second precision.
-			$date      = wc_string_to_datetime( date( 'Y-m-d', strtotime( $date ) ) );
+			$date      = wc_string_to_datetime( gmdate( 'Y-m-d', strtotime( $date ) ) );
```

Comment:
I'd rather not change the `date` to `gmdate` here unless there is a specific reason for doing so. This is not a PHPCS change and it may have unintended consequences

Reply:
Thanks for the review. I have reverted the `gmdate()` to `date()` in PR.